db/seeds.rb

```diff
@@ -5,3 +5,33 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+# announcements = Program.create!(
+#   name: "ANNOUNCEMENTS",
+#   opt_in_keywords: {en: ["STARTANNOUNCEMENTS"], es: ["STARTANNOUNCEMENTSSPANISH"]},
+#   opt_out_keywords: {en: ["STOPANNOUNCEMENTS"], es: ["ALTOANNOUNCEMENTS"]},
+#   opt_in_response: "Hello",
+#   opt_out_response: "Goodbye"
+# )
+# Mobility.with_locale(:es) do
+#   announcements.opt_in_response = "Hola"
+#   announcements.opt_out_response = "Adios"
+# end
+# announcements.save
+
+snap = Program.create!(
+  name: "SNAP",
+  opt_in_keywords: {
+    en: %w(START UNSTOP STARTSNAP),
+    es: %w(COMENZAR COMENZARSNAP)},
+  opt_out_keywords: {
+    en: %w(STOP END CANCEL UNSUBSCRIBE QUIT STOPSNAP),
+    es: %w(DETENER FIN CANCELAR SUSCRIBIRSE DEJAR ALTOSNAP)},
+  opt_in_response: "Hello",
+  opt_out_response: "You have successfully unsubscribed from DSS reminders. You will not receive any more messages from this number. Reply START to resubscribe."
+)
+Mobility.with_locale(:es) do
+  snap.opt_in_response = "Hola"
```

Comment:
Why is this set after creation as opposed to initialized for multiple locales like `opt_in_keywords` and `opt_out_keywords`?

Reply:
Made these consistent as discussed on zoom today